server/bin/org_migrator/org_migrator.py

```diff
@@ -64,7 +64,7 @@ def list_orgs(db):
 
 def jsonify(data):
     def data_converter(obj):
-        if isinstance(obj, bytearray) or isinstance(obj, buffer):
+        if isinstance(obj, (bytearray, memoryview)):
```

Comment:
In line with the other Python2 compat change, can we keep the buffer type present? I've also added some warning output and a handler in the event the type isn't present:

```
    def data_converter(obj):
        if isinstance(obj, (bytearray, buffer, memoryview)):
            return binascii.b2a_base64(obj)
        if isinstance(obj, datetime.datetime):
            return str(obj)

        log.warn("Unexpected type received; attempting to convert to string: %s" % (type(obj)))
        return str(obj)
```